### PR TITLE
Theme updates

### PR DESCRIPTION
**Pull request by starryeyez024**

### CODE REVIEW

File: web/css/json-editor.css [hunk 1]
```diff
@@ -156,6 +156,9 @@ body { -webkit-animation: androidBugfix infinite 1s; }
 }
 
 /* Styles specific to the schema content */
+.content {
```
Comment (castastrophe) [inline web/css/json-editor.css:159]: @starryeyez024 Where is the class content coming from?  I don't see it in any of the templates so I'm having a hard time finding an example to test that this works.

Comment (castastrophe) [reply web/css/json-editor.css:159]: bump

Comment (castastrophe) [reply web/css/json-editor.css:159]: @starryeyez024 double bump

Comment (starryeyez024) [reply web/css/json-editor.css:159]: I can't find this either. Sorry for the trouble, going to close this out.

**Pull request closed without merging**